dvc/utils/collections.py

```diff
@@ -98,7 +98,7 @@ def merge_dicts(src: Dict, to_update: Dict) -> Dict:
     return src
 
 
-def ensure_list(item: Union[Iterable[str], str, None]) -> List[str]:
+def ensure_list(item: Any) -> List[Any]:
```

Comment:
This is a bit tricky, it's used both for `targets` and for `paths`. It would have to be something like

```python
def ensure_list(
    item: Union[Iterable[str], Iterable[Optional[str]], str, None]
) -> List[Optional[str]]:
    ...
```

but this requires changing code in other contexts. Open to suggestions